mint.json

```diff
@@ -157,7 +157,7 @@
   ],
   "footerSocials": {
     "github": "https://github.com/traceloop",
-    "twitter": "https://twitter.com/traceloopdev"
+    "x-twitter": "https://x.com/traceloopdev"
```

Comment:
I think it's just `x`

https://mintlify.com/docs/settings/global

Reply:
Sorry but yes its just 'x'.